### PR TITLE
Fix chain

### PR DESCRIPTION
**Pull request by turuslan**
- fix chain methods
- fix multisig swap
- simplify config
- mainnet genesis and drand config
- payment channel address matcher

### CODE REVIEW

File: core/storage/mpool/mpool.cpp [hunk 1]
```diff
@@ -88,7 +90,14 @@ namespace fc::storage::mpool {
       if (apply.receipt.exit_code != vm::VMExitCode::kOk) {
         return apply.receipt.exit_code;
       }
-      // TODO: paych.collect
+      if (msg.method
+          == vm::actor::builtin::v0::payment_channel::Collect::Number) {
+        auto matcher{vm::toolchain::Toolchain::createAddressMatcher(
+            vm::version::getNetworkVersion(head->height()))};
+        if (matcher->isPaymentChannelActor(actor.code)) {
+          apply.receipt.gas_used += 76000;
```
Comment (Alexey-N-Chernyshov) [inline core/storage/mpool/mpool.cpp:98]: Please, add constant for 76000.

Comment (Alexey-N-Chernyshov) [reply core/storage/mpool/mpool.cpp:98]: It will be great if you can add a reference where the number comes from.

**Review by Alexey-N-Chernyshov — APPROVED** [review]

File: core/storage/mpool/mpool.cpp (hunk 1 above)
Comment (Alexey-N-Chernyshov) [reply core/storage/mpool/mpool.cpp:98]: https://github.com/filecoin-project/lotus/blob/191a05da4872bf9849f178e6db5c0d6e87d05baa/node/impl/full/gas.go#L281

Comment (turuslan) [reply core/storage/mpool/mpool.cpp]: Fixed

File: core/vm/actor/builtin/v0/multisig/multisig_actor.cpp [hunk 2]
```diff
@@ -260,7 +260,10 @@ namespace fc::vm::actor::builtin::v0::multisig {
       ABORT(VMExitCode::kErrIllegalArgument);
     }
 
-    std::replace(state.signers.begin(), state.signers.end(), from, to);
+    state.signers.erase(
+        std::remove(state.signers.begin(), state.signers.end(), from),
+        state.signers.end());
+    state.signers.push_back(to);
```
Comment (wer1st) [inline core/vm/actor/builtin/v0/multisig/multisig_actor.cpp:266]: What was wrong? Order?

**Review by wer1st — APPROVED** [review]

File: core/vm/actor/builtin/v0/multisig/multisig_actor.cpp (hunk 2 above)
Comment (turuslan) [reply core/vm/actor/builtin/v0/multisig/multisig_actor.cpp:266]: Yes, https://github.com/filecoin-project/specs-actors/blob/v2.3.3/actors/builtin/multisig/multisig_actor.go#L415-L422

**Pull request merged**